### DISCUSSION TITLE
colors in google colab notebooks

### DISCUSSION THREAD

**Discussion by galtay** [Textualize/rich#2275] [Q&A]
Hello rich users! 

I have so much fun using this package and its led me to try it everywhere I write Python. Was wondering if anyone has had issues with the color availability when using rich in google colab notebooks? I know this is probably a pretty niche use case, but was curious. Running `python -m rich` shows a pretty blotchy color space in the `Colors` section and the colors that do get loaded are hard to read on the light grey dark mode background. 

I can't tell if the colab notebook is not rendering the colors correctly or if the default color choices just happen to be a bad fit for grey background of the notebooks dark mode. Are there different pre-built themes in rich to try out? Attaching a link to a colab notebook below and a screenshot. 

Either way, excellent tool and so much fun to use! 

https://colab.research.google.com/drive/1UNvRCB59cD6yuR7wlOSk8QjPxL3uosjo?usp=sharing

![rich_colab](https://user-images.githubusercontent.com/663051/168442171-197814f1-407e-4012-8471-b7e71d0b1aeb.png)

**Comment by Nneji123** [comment]
This looks great going to try this on my next colab project and see how it turns out
